python/torch_mlir/extras/fx_importer.py

```diff
@@ -207,10 +207,35 @@
 }
 
 
-"""Check whether an object in our graph is symbolic"""
+def sparsity_encoding(shape: torch.Size, sparse_layout : torch.layout) -> str:
+   """Returns sparse tensor encoding for the given sparse layout as string.
+
+   Note that we currently rely on string parsing to construct the MLIR tensor
+   type (both for dense as well as sparse). In the long run this should be
+   replaced by proper IR building, obviously.
```

Comment:
We can use upstream apis but we resort to string pasting for torch APIs because torch-mlir doesn't expose sufficient python APIs for all of this -- and doesn't have a super good way to export them to downstreams even if it did.

Reply:
Ah, I will remove the comment in that case (given that it is uncertain we will change that soon)